media/js/mozorg/contact-spaces.js

```diff
@@ -160,6 +160,7 @@
                     // Hide spaces and community nav+meta
                     mozMap.toggleNav('contact');
                     mozMap.toggleMobileNav('contact');
+                    mozMap.setMapState();
```

Comment:
Yes we have functional tests - I don't think it is unstable per-say, just something isn't being set quite right. The contact tab should need to follow the same logic as the other two tabs.